org.supertuxproject.SuperTux.json

```diff
@@ -96,6 +96,11 @@
                 "-DCMAKE_BUILD_TYPE=Release",
                 "-DIS_SUPERTUX_RELEASE=ON"
             ],
+            "post-install": [
+                "install -Dm644 updated-appdata.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml",
+                "rsvg-convert supertux2.svg -w 128 -h 128 -a -f png -o 128.png",
```

Comment:
No idea, if this fixes your problem, but I would recommend to only copy the svg over as an icon and don't convert - as it's much more futureproof.

```
install -Dm644 supertux2.svg /app/share/icons/hicolor/scalable/apps/org.supertuxproject.SuperTux.svg
```

Should hopefully work

Reply:
you don't need this. a SVG is enough

Reply:
But that needs to be in some folder?

Reply:
this should be removed, its unecessary.